cmd/invoke.go

```diff
@@ -44,10 +44,11 @@ var InvokeCmd = &cobra.Command{
 }
 
 func init() {
-	InvokeCmd.Flags().StringVarP(&invokeAppID, "app-id", "a", "", "the app id to invoke")
-	InvokeCmd.Flags().StringVarP(&invokeAppMethod, "method", "m", "", "the method to invoke")
-	InvokeCmd.Flags().StringVarP(&invokePayload, "payload", "p", "", "(optional) a json payload")
-	InvokeCmd.Flags().StringVarP(&invokeVerb, "verb", "v", defaultHTTPVerb, "(optional) The HTTP verb to use. default is POST")
+	InvokeCmd.Flags().StringVarP(&invokeAppID, "app-id", "a", "", "The application's id to invoked")
```

Comment:
"The application id to invoke" 
and 
"The method to invoke"
Have capital for "The". In fact can you check that all sentences start with capital. for example most of the help sentences are wrong like this one
"help for init" should be "Help for init". Check all of them.

Reply:
Fixed app and method.

I will double check but everything was converted to start with capital letter.